simulator/kruxsim/mocks/lcd.py

```diff
@@ -411,6 +412,7 @@ def run():
         x = width() - w - x
     radius = min(radius, min(w, h) // 2)
     pg.event.post(pg.event.Event(events.LCD_FILL_RECTANGLE_EVENT, {"f": run}))
+    time.sleep(0.001)
```

Comment:
`0.001` froze my simulator on first click and logged the follow message on terminal:

```bash
Exception in thread Thread-1 (run_krux):
```

With a bigger value was possible:

```suggestion
    time.sleep(0.01)
```

Reply:
Can you test again plz? I did use another approach because yours could lag Tinyseed